dashboard/app/models/user_level.rb

```diff
@@ -166,8 +166,8 @@ def self.update_lockable_state(user_id, level_id, script_id, locked, readonly_an
     # no need to create a level if it's just going to be locked
     return if !user_level.persisted? && locked
 
+    # Used to set as submitted if locked and readonly, removing that 01/21
     user_level.assign_attributes(
-      submitted: locked || readonly_answers,
```

Comment:
let's confirm with Amanda whether we should remove this entirely or still set it to `submitted` when `readonly_answers == true`.

Reply:
Circling back- Amanda confirms okay to remove entirely. Resolving now